pkg/operator/ceph/config/config.go

```diff
@@ -132,77 +128,3 @@ func SetDefaultAndUserConfigs(
 
 	return nil
 }
-
-// LegacyConfigMapOverrideDefined checks to see if the legacy ConfigMap override exists with
-// overrides defined. If it exists and has config overrides defined, this will return true. If it
-// does not exist or exists without config overrides set, it will return false.
-func LegacyConfigMapOverrideDefined(context *clusterd.Context, namespace string) (bool, error) {
```

Comment:
@BlaineEXE These legacy overrides aren't needed anymore, right?

Reply:
Correct. I must have missed removing this.